plugins/node/opentelemetry-instrumentation-mongodb/package.json

```diff
@@ -58,7 +58,7 @@
     "@types/mongodb": "3.6.20",
     "@types/node": "18.6.5",
     "mocha": "7.2.0",
-    "mongodb": "3.6.11",
+    "mongodb": "^6.4.0",
```

Comment:
If this is done, then the default `"test"` script will need to change above, right?